SwiftVideoBackground/MoreViewController.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+class MoreViewController: UIViewController {
+    @IBOutlet weak var view1: UIView!
```

Comment:
Can we add this here?:

```swift
override var prefersStatusBarHidden: Bool { return true }
```

Reply:
The MoreViewController doesn't play a full screen video. Do you still want the status bar hidden?

Reply:
In that case, I think the status bar should be fine

Reply:
✅